examples/step-44/step-44.cc

```diff
@@ -821,13 +821,10 @@ namespace Step44
     // object (see the @ref threads module for more information on this).
     //
     // We declare such structures for the computation of tangent (stiffness)
-    // matrix, right hand side, static condensation, and for updating
+    // matrix and right hand side vector, static condensation, and for updating
     // quadrature points:
-    struct PerTaskData_K;
-    struct ScratchData_K;
-
-    struct PerTaskData_RHS;
-    struct ScratchData_RHS;
+    struct PerTaskData_ASM;
```

Comment:
Maybe we can call this ``PerTaskData`` now?

Reply:
You can't see it in the diff, but within the next few lines another two PerTaskData structs (PerTaskData_SC and PerTaskData_UQPH) are defined. So I think that it makes sense to keep the naming as is to better distinguish between them.